config/config.go

```diff
@@ -35,41 +25,12 @@ import (
 // Parse parses a config of any supported version and returns the equivalent config at the latest
 // supported version.
 func Parse(raw []byte) (types_exp.Config, report.Report, error) {
-	version, r, err := util.GetConfigVersion(raw)
-	if err != nil {
-		return types_exp.Config{}, r, err
-	}
-	switch version {
-	case types_exp.MaxVersion:
-		return v3_3_experimental.Parse(raw)
-	case types_3_2.MaxVersion:
-		return exp_from_3_2(v3_2.Parse(raw))
-	case types_3_1.MaxVersion:
-		return exp_from_3_2(v3_2_from_3_1(v3_1.Parse(raw)))
-	case types_3_0.MaxVersion:
-		return exp_from_3_2(v3_2_from_3_1(v3_1_from_3_0(v3_0.Parse(raw))))
-	default:
-		return types_exp.Config{}, report.Report{}, errors.ErrUnknownVersion
-	}
-}
-
-func exp_from_3_2(cfg types_3_2.Config, r report.Report, err error) (types_exp.Config, report.Report, error) {
-	if err != nil {
-		return types_exp.Config{}, r, err
+	if len(raw) == 0 {
+		return types_exp.Config{}, report.Report{}, errors.ErrEmpty
 	}
-	return trans_exp.Translate(cfg), r, nil
-}
-
-func v3_2_from_3_1(cfg types_3_1.Config, r report.Report, err error) (types_3_2.Config, report.Report, error) {
+	config, r, err := v3_3_experimental.ParseCompatibleVersion(raw)
```

Comment:
In every case, we should just pass through the results we got from `ParseCompatibleVersion`.  So this can just be

```
return v3_3_experimental.ParseCompatibleVersion(raw)
```